client/src/Movies/MovieList.js

```diff
@@ -32,9 +34,14 @@ export default class MovieList extends Component {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
+  const { title, director, metascore, stars, id } = movie;
```

Comment:
nice job here,  but passing in id, you dont have to reference movie.id in the path.